config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -88,6 +88,7 @@
         - --gcp-zone=us-west1-b
         - --parallelism=1
         - '--label-filter={{label_filter}}'
+        - --skip= # Override kubetest2 default in https://github.com/kubernetes-sigs/kubetest2/blob/9f385d26316f5256755bb8fe333970aa5759ec7f/pkg/testers/node/node.go#L92
```

Comment:
Yeah, been there, seen that: https://github.com/kubernetes/test-infra/pull/34005 https://github.com/kubernetes/test-infra/pull/34022

Just wondering why they've changed defaults for this option.

/lgtm

Reply:
Yes, very unexpected. It probably made sense to people who knew that they wanted that.